android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -238,6 +241,49 @@ class TranslationsMiddleware(
         )
     }
 
+    /**
+     * Retrieves the list of downloaded languages using [scope] and dispatches the result to the
```

Comment:
nit: 
```suggestion
     * Retrieves the list of downloaded languages and dispatches the result to the
```

(That was a stale comment error that didn't get refactored out on the earlier similar functions ones.)

Reply:
Maybe also expand here that when it isn't a tab it is:
`EngineExceptionAction` and when it is a tab it is both `EngineExceptionAction` and `TranslateExceptionAction`?

Reply:
Updated the doc comment, thanks!